browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -65,12 +65,17 @@ class MarkdownNoteDetail extends React.Component {
   findTitle (value) {
     let splitted = value.split('\n')
     let title = null
+    let isMarkdownInCode = false
 
     for (let i = 0; i < splitted.length; i++) {
       let trimmedLine = splitted[i].trim()
-      if (trimmedLine.match(/^# .+/)) {
-        title = trimmedLine.substring(1, trimmedLine.length).trim()
-        break
+      if (trimmedLine.match('```')){
+        isMarkdownInCode = !isMarkdownInCode
+      } else {
+        if(isMarkdownInCode === false && trimmedLine.match(/^# +/)) {
+          title = trimmedLine.substring(1, trimmedLine.length).trim()
+          break
+        }
       }
```

Comment:
Thanks for corrections! And I recognized 1 point. You can reduce 1 indent like this below:

```
if (trimmedLine.match('```')){
  isMarkdownInCode = !isMarkdownInCode
} else if(isMarkdownInCode === false && trimmedLine.match(/^# +/)) {
  title = trimmedLine.substring(1, trimmedLine.length).trim()
  break
}
```